messages/fr.json

```
@@ -1,63 +1,63 @@
 {
-  "pickupPointsModal.loading": "Chargement…",
-  "pickupPointsModal.deliver": "Délivrer",
-  "pickupPointsModal.pickup": "Retrait",
+  "pickupPointsModal.loading": "Chargement en cours…",
+  "pickupPointsModal.deliver": "Livrée",
+  "pickupPointsModal.pickup": "Relais colis",
   "pickupPointsModal.items": "{itemsAmount, plural, =0 {} one {# article} other {# articles}}",
-  "pickupPointsModal.cancelBackList": "Voir tous les points de retrait",
+  "pickupPointsModal.cancelBackList": "Voir les relais colis",
   "pickupPointsModal.slaOption": "{name} - {price} - {shippingEstimate}",
   "pickupPointsModal.free": "Gratuit",
-  "pickupPointsModal.selectPickupPoint": "Sélectionnez un point de retrait",
-  "pickupPointsModal.pointDetails": "Détails du point de retrait",
+  "pickupPointsModal.selectPickupPoint": "Choisir un relais colis",
+  "pickupPointsModal.pointDetails": "Détails du relais colis",
   "pickupPointsModal.closeButton": "fermer",
   "pickupPointsModal.list": "Liste",
   "pickupPointsModal.map": "Carte",
   "pickupPointsModal.pickupPoint.modify": "Modifier",
-  "pickupPointsModal.details": "Voir les détails ou changer le point de retrait.",
-  "pickupPointsModal.searchLocationMap": "Recherche de rue, code postal, quartier ou ville",
-  "pickupPointsModal.productsInPoint": "Disponibilité du produit dans ce point de retrait",
+  "pickupPointsModal.details": "Voir les détails ou modifier le relais colis.",
+  "pickupPointsModal.searchLocationMap": "Rechercher une rue, un code postal, un quartier ou une ville",
+  "pickupPointsModal.productsInPoint": "Produits disponibles dans ce relais colis",
   "pickupPointsModal.aditionalInfo": "Autres informations",
-  "pickupPointsModal.unavailable": "indisponible",
-  "pickupPointsModal.unavailableItemsAmount": "{itemsAmount, plural, =0 {} one {# article non disponible} other {# articles non disponible}}",
-  "pickupPointsModal.allItemsAvailable": "Tous les produits disponibles",
-  "pickupPointsModal.noneItemsAvailable": "Aucun produit disponible",
-  "pickupPointsModal.businessHours": "Horaire commercial",
-  "pickupPointsModal.confirmPoint": "Retrait à ce point",
-  "pickupPointsModal.price": "{value, plural, =0 {Gratuit} other {{formattedPrice}}}",
-  "pickupPointsModal.askGeolocation": "Rechercher une adresse avec votre emplacement",
+  "pickupPointsModal.unavailable": "non disponible",
+  "pickupPointsModal.unavailableItemsAmount": "{itemsAmount, plural, =0 {} one {#article non disponible} other {# article non disponible}}",
```

I think we need this blank space after the `#`, right?

```suggestion
  "pickupPointsModal.unavailableItemsAmount": "{itemsAmount, plural, =0 {} one {# article non disponible} other {# article non disponible}}",
```